instructions/level1.md

```diff
@@ -329,10 +347,14 @@ To validate our form we will need to follow next steps:
   that we create for you in our `index.css` file. It only makes the border of the
   field that has this class red.
    
-  TODO: 1.We will need to use `querySelector` to target the element on the page. 
-  And all of it we can do inside our forEach loop. So we are targeting the whole 
-  form and then fields with specific name (that our key will represent 
-  `[name="${key}"]`).
+  TODO: 1.We will need to use `document.querySelector` to target the element on
+  the page. And all of it we can do inside our forEach loop. So we are targeting
+  the whole form and then fields with specific name (that our key will represent 
+  `[name="${key}"]`). This will look like:
+
+  ```
+  document.querySelector(`[name="${key}"]`)
```

Comment:
Because this is using es6 string interpolation, which uses backticks, I've done this as a block statement.